.github/workflows/ci-wheels.yml

```diff
@@ -27,25 +27,28 @@ concurrency:
 jobs:
   build_bdist:
     name: "Build ${{ matrix.os }} (${{ matrix.arch }}) wheels"
-    runs-on: ${{ matrix.os }}-latest
+    runs-on: ${{ matrix.os }}
     strategy:
       fail-fast: false
       matrix:
         # TBD: extend to support windows
-        os: ["ubuntu", "macos"]
-        arch: ["x86_64", "arm64"]
-        exclude:
-        - os: ubuntu
-          arch: arm64
+        os: ["ubuntu-latest", "macos-11", "macos-14"]
         include:
-        - os: ubuntu
+        - os: ubuntu-latest
+          arch: x86_64
           incdir: /usr/include/udunits2
           libdir: /usr/lib64
           xml_path: /usr/share/udunits/udunits2.xml
-        - os: macos
+        - os: macos-11
+          arch: x86_64
           incdir: /usr/local/Cellar/udunits/2.2.28/include
           libdir: /usr/local/Cellar/udunits/2.2.28/lib
           xml_path: /usr/local/Cellar/udunits/2.2.28/share/udunits/udunits2-common.xml
+        - os: macos-14
+          arch: arm64
+          incdir: /opt/homebrew/Cellar/udunits/2.2.28/include
+          libdir: /opt/homebrew/Cellar/udunits/2.2.28/lib
+          xml_path: /opt/homebrew/Cellar/udunits/2.2.28/share/udunits/udunits2-common.xml
```

Comment:
I'm OK with this. [Here is where I needed to look it up](https://earthly.dev/blog/homebrew-on-m1/)

Reply:
Yup, also the traceback and diagnostics from `cibuildwheels` pretty much tells you too when you have the incorrect path prefix 👍